### ISSUE TITLE
Tax Name

### ISSUE THREAD

**Issue by davidsherlock** [easydigitaldownloads/easy-digital-downloads#5231]
The generic name "Tax" is used throughout EDD, however different jurisdictions have different names for local sales taxes such as VAT, GST, HST, etc.

When setting the tax rates, it would be useful to be able to set on a per-line basis the tax name along with the rate (WooCommerce has similar functionality). If a name was entered, this would override the default "Tax" name and appear at the checkout and on any applicable invoices.

This would help move the core of EDD to being VAT compliant out of the box.

**Comment by mihaijoldis** [comment]
Just had a request like this nohttps://secure.helpscout.net/conversation/1784653435/1782715?folderId=5747299
Customer seems to recall us having an option to rename Tax to GST

**Comment by cklosowski** [comment]
I don't think we ever did have this, our fees can have names, but this is something we could for sure look at adding. I'm going to remove it from 3.1, and let the new tax system bake a little bit before we start making column changes to the adjustments table.
